hw1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Jersey</title>
+</head>
+<body>
+    <div class="weather">
+        <div class="weather__top">
+            <div class="wt__item">
+                <img src="images/3.png">
+                <div class="wt__refresh"><img src="images/refresh.png"></div> 
+            </div>
+        </div>
+
+        <div class="weather__middle">
+          <div class="wm__item">
+            <div class="wm__degrees">79</div>
+            <div class="o">o</div>
+        </div>
+        <div class="wm__item_middle">
+            <div class="wm__weather">Cloudy Skies</div>
+            <div class="wm__location">Sicklerville, New Jersey </div>
+        </div>
+        <div class="wm__lastitem">
+            <div class="wm__month">MAY</div>
+            <div class="wm__date">21</div>
+        </div>
+    </div>
+
+    <div class="weather_bottom">
+        <div class="wb__parameters">
+           <div class="wb__item">
+            <div class="wb__icon"><img src="images/wind.png" alt=""></div>
+            <div class="wb__text">2 MPH</div>
+        </div>
+        <div class="wb__item">
+            <div class="wb__icon"><img src="images/rain.png" alt=""></div>
+            <div class="wb__text">33%</div>
+        </div>
+        <div class="wb__item">
+            <div class="wb__icon"><img src="images/sun.png" alt=""></div>
+            <div class="wb__text">83%</div>
+        </div>
+    </div>
+    <div class="wb__scroll">
+     <div class="scroll__selected"></div>
+     <div class="scroll"></div>
+     <div class="scroll"></div>
+ </div>
+
+</div>
+</div> 
+</body>
```

Comment:
Please fix the identation, you could use prettier for keeping your code readability.
https://prettier.io/docs/en/index.html